src/target/riscv/program.c

```diff
@@ -121,14 +121,11 @@ int riscv_program_csrw(struct riscv_program *p, enum gdb_regno s, enum gdb_regno
 	return riscv_program_insert(p, csrrw(GDB_REGNO_ZERO, s, csr - GDB_REGNO_CSR0));
 }
 
-int riscv_program_fence_i(struct riscv_program *p)
+int riscv_program_fences(struct riscv_program *p)
 {
-	return riscv_program_insert(p, fence_i());
-}
-
-int riscv_program_fence(struct riscv_program *p)
-{
-	return riscv_program_insert(p, fence());
+	if (riscv_program_insert(p, fence()) != ERROR_OK) return ERROR_FAIL;
```

Comment:
I believe the OpenOCD style guide does not allow for putting `return` on the same line is `if`. You can check with tools/checkpatch.sh (although that will only check the last git commit, and exit early until you add a Signed-off line).